simulation/array_simulator.pro

```diff
@@ -51,7 +51,27 @@ PRO array_simulator,vis_arr,vis_weights,obs,status_str,psf,params,jones,error=er
   IF ~silent THEN print,'Calculating beam model'
   psf=beam_setup(obs,status_str,file_path_fhd=file_path_fhd,restore_last=0,silent=silent,timing=t_beam,no_save=0,_Extra=extra)
   IF Keyword_Set(t_beam) THEN IF ~silent THEN print,'Beam modeling time: ',t_beam
-  
+ 
+  ;; Get a selection radius based on the primary beam width  TODO --- Only do this if include_catalog_sources == 0, in which case get it from generate_source_cal_list
+  if not keyword_set(beam_threshold) then beam_threshold = 0.05
+  if keyword_set(allow_sidelobe_sources) THEN beam_threshold = 0.01
```

Comment:
This is not a good place to set these defaults. Based on the code below, these lines should simply be removed from this file.